.github/workflows/next-lint-check.yml

```diff
@@ -0,0 +1,31 @@
+name: Next.js Linting Check
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Checkout [this convention](https://github.com/asyncapi/conference-website/blob/6c6ab9074d49d2f4f99e498433f6a369ceebbf68/.github/workflows/lint-pr-title.yml#L6C1-L8C69) in the asyncapi org regarding pull request

Reply:
thanks for the recommendations @AnimeshKumar923  I just used a template there were going to be these changes, but thanks for the review!

Reply:
> Checkout [this convention](https://github.com/asyncapi/conference-website/blob/6c6ab9074d49d2f4f99e498433f6a369ceebbf68/.github/workflows/lint-pr-title.yml#L6C1-L8C69) in the asyncapi org regarding pull request

I get it but I dont think a linting workflow will require so many activations?

Reply:
Yes, I guess it will work as you're expecting it. For every new commit it will trigger the workflow for the linting so I guess it fine to leave as it is. I was just suggesting the convention. Let's have it later on in the future. :+1: